src/index.ts

```diff
@@ -309,7 +309,8 @@ export const WebSockets = fmt(_WebSockets)
 
 const _WebSocketsSecure = or(
   and(_WEB, literal('wss'), optional(peerId())),
-  and(_WEB, literal('tls'), literal('ws'), optional(peerId()))
+  and(_WEB, literal('tls'), literal('ws'), optional(peerId())),
+  and(_TCP, literal('tls'), literal('sni'), string(), literal('ws'), optional(peerId()))
```

Comment:
This can be added to the existing rule:

```suggestion
  and(_WEB, literal('tls'), optional(literal('sni'), string()), literal('ws'), optional(peerId()))
```